plugin-gradle/README.md

```diff
@@ -139,10 +139,10 @@ To apply freshmark to all of the `.md` files in your project, with all of your p
 ```gradle
 spotless {
 	freshmark {
-		target 'README.md', 'CONTRIBUTING.md'	// defaults to '**/*.md'
-		propertiesFile('gradle.properties')		// loads all the properties in the given file
+		target 'README.md', 'CONTRIBUTING.md'			// defaults to '**/*.md'
+		propertiesFile 'fresh.xml', 'mark.properties'	// loads all the properties in the given files
 		properties {
-			it.put('key', 'value')				// specify other properties manually
+			it.put('key', 'value')						// specify other properties manually
```

Comment:
Ok, too fast. Markdown Preview Plus is using mono-spaced fonts (was actually surprised by the result), but I of course have to do a conversion with a tool that uses mono-spaced fonts too.
Don't know, in the end I think it would look more pretty with spaces. I would propose an indentation of three.
If you like, I can refactor the file.